src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/ParamTaglet.java

```diff
@@ -225,23 +200,15 @@ private Content getInheritedTagletOutput(ParamKind kind, Element holder,
      *
      * @param paramTags the array of {@code @param DocTree} to convert.
      * @param writer the TagletWriter that will write this tag.
-     * @param alreadyDocumented the set of exceptions that have already
-     *        been documented.
-     * @param rankMap a {@link java.util.Map} which holds ordering
```

Comment:
It's ironical that a method whose job, at least in part, is to warn about duplicated `@param` tags had one of its `@param` tags duplicated. Was it an easter egg of sorts?